Policies/github-subprocessors-and-cookies.md

```diff
@@ -0,0 +1,110 @@
+---
+title: GitHub Subprocessors and Cookies
+redirect_from:
+ - /subprocessors/
+ - /github-subprocessors/
+ - /github-tracking/
+ - /github-cookies/
+---
+
+Effective date: **April 19, 2018**
+
+GitHub provides a great deal of transparency regarding how we use your data, how we collect your data, and with whom we share your data. To that end, we provide this page, which details [our subprocessors](#github-subprocessors), how we use [cookies](#cookies-on-github), and where and how we perform any [tracking on GitHub](#tracking-on-github).
+
+### GitHub Subprocessors 
+
+When we share your information with third party subprocessors, such as our vendors and service providers, we remain responsible for it. We work very hard to maintain your trust when we bring on new vendors, and we require all vendors to enter into data protection agreements with us that restrict their processing of Users' Personal Information (as defined in the [Privacy Statement](/articles/github-privacy-statement/). 
+
+| Name of Subprocessor | Description of Processing | Location of Processing |
+|---|---|---|
+| Box | Corporate document storage | United States |
+| Braintree (PayPal) | Subscription credit card payment processor | United States |
+| DocuSign | Contract signature processor | United States |
+| DropBox | Corporate document storage | United States |
+| Front | Support inbox system | United States |
+| Google Apps | Internal company infrastructure | United States |
+| Google Analytics | Website analytics and performance | United States |
+| Lithium | Community forum software provider | United States |
+| MailChimp | Customer ticketing mail services provider | United States |
+| Oracle | Corporate financial system | United States |
+| Salesforce.com | Customer relations management | United States |
+| Seal | Contract clause analysis system | United States |
+| ZenDesk | Customer support ticketing system | United States |
+| Zuora | Corporate billing system | United States |
+
+When we bring on a new vendor or other subprocessor who handles our Users' Personal Information, or remove a subprocessor, or we change how we use a subprocessor, we will update this page. 
+
+### Cookies on GitHub 
+
+GitHub uses cookies to make interactions with our service easy and meaningful. We use cookies (and similar technologies, like HTML5 localStorage) to keep you logged in, remember your preferences, and provide information for future development of GitHub.
```

Comment:
I guess all the `similar technologies` should be named, and for every one should be clearly stated if it is used or if it is hot used.

Reply:
@nsqe would you comment this?

Reply:
We've explained above, in the section on [Information from website browsers](https://help.github.com/articles/github-privacy-statement/#information-from-website-browsers), exactly what information we collect from visitors to the site via their browsers. We don't go down into collecting fonts and plug-ins, because we don't care. Live your best life.